### ISSUE TITLE
Talkback doesn't describe any items in the launcher

### ISSUE THREAD

**Issue by hedning** [OpenLauncherTeam/openlauncher#214]
#### General information

Hi, I was checking out how the android screen-reader works and noticed that it doesn't read anything (not even app icons) in OpenLauncher.

* **App version: 0.5.3**
* **Android Version: 7.0**

#### Expected result
Talkback should be able to describe what the user clicks on.

**What does happen instead?**
There's no talkback :(

#### Steps to reproduce
* Enable Talkback (Settings -> Accessibility -> Talkback)
* Tap on different icons in the launcher

**Comment by gsantner** [comment]
Do you have an idea how to work on this or what is needed to support this? @hedning

**Comment by hedning** [comment]
Not really, haven't worked on any android stuff, but I'm guessing it's simply missing labels: https://developer.android.com/guide/topics/ui/accessibility/apps.html#label-elements

**Comment by gsantner** [comment]
Can you see - in meaning of verify the status/if it's working - when somebody pushed changes about this?

**Comment by hedning** [comment]
It's easy to test yourself, just go to `settings -> accessibility -> talkback` and turn it on.

**Comment by gsantner** [comment]
I don't have such an option, and Talkback seems to be an Google (Play Services) application/service that may not be installed on all devices (e.g. on mine)

**Comment by davidbtc2009** [comment]
Google accessibility suite and google text to speech can be downloaded from the play store or the aurora store found on F-Droid. This is still an issue in v0.7.4 added on 12/31/2020

**Comment by gsantner** [comment]
If a test apk file is available for download can you do test and report back if it helps? I'm totally for improving accessibility but I'm not used to screenreaders or know anybody who does - it's much more important to get info from somebody who actually uses (or requires) it.

**Comment by davidbtc2009** [comment]
Talkback does read the apps when the app drawer is open but is not reading the icons that are placed on the home screen. Maybe start their. May be simple as some code coppy. If i could send a video i would.

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by gsantner** [comment]
I asked if you can help testing

**Comment by davidbtc2009** [comment]
Absolutely. As long as its an apk file. I am worthless at building anything from source

Sent from ProtonMail mobile

-------- Original Message --------
